### ISSUE TITLE
No Projects will Launch

### ISSUE THREAD

**Issue by 3DSki** [renpy/renpy#4878]
8.1.1 fails to launch its own tutorials (Tutorial or The Question) on latest Win 10.  I have Windows Defender turned off (some have found that that's an issue). There is no failure showing in log.txt: 

Sun Jul 30 15:48:32 2023
Windows-10-10.0.19045
Ren'Py 8.1.1.23060707
 

Bootstrap to the start of init.init. took 0.03s
Early init. took 0.00s
Loader init. took 0.03s
Loading error handling. took 0.18s
Loading script. took 0.37s
Loading save slot metadata. took 0.02s
Loading persistent. took 0.00s
Set script version to: None (alternate path)
 - Init at launcher/game/project.rpyc:786 took 0.29994 s.
Running init code. took 0.60s
Loading analysis data. took 0.01s
Analyze and compile ATL. took 0.00s
Reloading save slot metadata. took 0.01s
Index archives took 0.00s
Dump and make backups. took 0.00s
Cleaning cache. took 0.00s
Making clean stores. took 0.00s
Initial gc. took 0.03s
DPI scale factor: 1.250000
nvdrs: Loaded, about to disable thread optimizations.
nvdrs: Disabled thread optimizations.
Creating interface object. took 0.27s
Cleaning stores took 0.00s
Init translation took 0.03s
Build styles took 0.00s
Load screen analysis took 0.03s
Analyze screens took 0.00s
Save screen analysis took 0.00s
Prepare screens took 0.07s
Save pyanalysis. took 0.00s
Save bytecode. took 0.00s
Running _start took 0.00s
Performance test:
Interface start. took 0.07s

Initializing gl2 renderer:
primary display bounds: (0, 0, 1920, 1080)
swap interval: 1 frames
Windowed mode.
Vendor: "b'NVIDIA Corporation'"
Renderer: b'NVIDIA GeForce GTX 1650/PCIe/SSE2'
Version: b'4.6.0 NVIDIA 517.00'
Display Info: None
Screen sizes: virtual=(800, 600) physical=(1000, 750) drawable=(1000, 750)
Maximum texture size: 4096x4096

I have no issues with launching The Question with Ren'Py 7.6, but I prefer to start with Python 3.

EDIT:  SOLVED - I thought I had removed PYTHON environment variables, but hadn't. Once I remove them RenPy successfully launched the provided Tutorial and The Question games.

**Comment by renpytom** [comment]
The only thing I can think of is potentially unrespresentable characters in the path to the game. Try moving Ren'Py into c:\renpy-8.1.1-sdk , and see if that helps the issue.

**Comment by 3DSki** [comment]
The current location was directly of E: (a large laptop USB drive). I
copied that to C: as you suggested and the failure is the same. I've tried
multiple versions of 8.x. RenPy 7.6 works fine. I had a version of Python
3.11.4 and I even uninstalled that, along with the PYTHON related env
settings. This 8.1.1. install is on a pretty modern laptop that's only a
couple years old. I can try by older Win 10 desktop.

**Comment by 3DSki** [comment]
I had Python installed, but thought I removed it and environment variables.
The environment variables were still there. Once I deleted those variables
RenPy launches were successful.

There was some process I was performing that seemed to need
PYTHONHOME/PYTHONBIN, so I had manually added those. I'm pretty sure I'm
going to be reinstalling Python 3, so I'll have to be careful with those
environment variables if have to set them again.
